Backpack-SwiftUI/Icons/Classes/Generated/BPKIcons.swift

```diff
@@ -27,6 +27,7 @@ public extension BPKIcon {
     static let account = BPKIcon(name: "account")
     static let addCircle = BPKIcon(name: "add-circle")
     static let adult = BPKIcon(name: "adult")
+    static let ai = BPKIcon(name: "ai")
```

Comment:
How's the linter not complaining about a 2 character length name?

Reply:
ai ai ai. Let me check D:

Reply:
I will raise a separate PR to resolve linting issues 😬